README.md

```diff
@@ -22,16 +24,23 @@ See [action.yml](action.yml)
     # Default: 'space-delimited'
     format: ''
     # Filter files using a regex
-    filter: '*'
+    glob-filter: '*'
```

Comment:
Same here, the verison of the action should be changed too right?

Reply:
it should. `yarn version --no-git-tag-version --major`

Reply:
right, @micalevisk, @Ana06, should we also change the repository URLs in the package.json? Or are we planning on trying to merge in the original repo in the future?

Reply:
I did the yarn command thingie, I'll let you all check :)

Reply:
should we call it `grob-filter` or just filter? I like `filter`, as we don't have any other filer I think the name doesn't need to be more specific as long as we document it properly. But I don't have a strong opinion... 🤔 :thinking: Any other opinion?

Reply:
@rnsc 
>should we also change the repository URLs in the package.json? Or are we planning on trying to merge in the original repo in the future? 

I think we can change the urls. I was hopping upstream integrates these changes when I originally forked it, but it doesn't look like it will happen.